test/repository_test.rb

```diff
@@ -163,6 +182,45 @@
         it 'raises an error when not persisted' do
           -> { UserRepository.update(user2) }.must_raise(Lotus::Model::NonPersistedEntityError)
         end
+
+        describe 'when entity contain updated_at without created_at' do
+          before do
+            CommentRepository.adapter    = adapter.new(mapper, uri)
+
+            CommentRepository.collection = :comments
+          end
+
+          after do
+            CommentRepository.clear
+          end
+
+          let(:comment) { CommentRepository.create(Comment.new(comment: 'So great')) }
+
+          it 'not respond_to created_at' do
+            comment.wont_respond_to(:created_at)
```

Comment:
amend this test, we are not testing if comment responding to created_at or not, but asserting against if updated_at is touched or not